Casks/m/macwhisper.rb

```diff
@@ -1,8 +1,8 @@
 cask "macwhisper" do
-  version "12.18.3,1293"
-  sha256 "4449af9ef2d7c6f2cce916fddb4e37f09673275fefc678d6af6251c2fc6fa4d9"
+  version "12.18.3,1293,B"
+  sha256 :no_check
```

Comment:
```suggestion
  sha256 "e9d41224e6e379fc50202cb95919991f209ce0ea87aaa4d3d27e464efd5be165"
```

We should be able to use a checksum.